code/modules/mob/living/status_procs.dm

```diff
@@ -338,6 +338,27 @@
 		S = apply_status_effect(STATUS_EFFECT_ADMINSLEEP)
 	return S
 
+/////////////////////////////////// STASIS ////////////////////////////////////
+
+/mob/living/proc/IsStasis()
+	return has_status_effect(STATUS_EFFECT_STASIS)
+
+/mob/living/proc/ToggleStasis()
+	var/datum/status_effect/incapacitating/stasis/S = IsStasis()
```

Comment:
Why not use .

Reply:
Should I? I was going by how the other status effects are currently structured